examples/Tutorial_physicell.ipynb

```diff
@@ -79,7 +82,7 @@
     "\n",
     "example_data = PhysicellData(\n",
     "    box_size=np.array([box_size, box_size, 100.0]),\n",
-    "    timestep=360.0,  # seconds\n",
+    "    timestep=360000.0,\n",
```

Comment:
Is this supposed to represent 360 seconds (360000 ms)? Then shouldn't it be `timestep=360.0`. since we have the `time_unit_factor_seconds` specified as 1e-3?